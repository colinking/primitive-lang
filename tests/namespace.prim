// SYNTAX TEST "source.prim" "just namespace"

namespace testABC123
