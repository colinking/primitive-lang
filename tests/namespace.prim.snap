>// SYNTAX TEST "source.prim" "just namespace"
#^^ source.prim comment.line.double-slash.prim punctuation.definition.comment.prim
#  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.prim comment.line.double-slash.prim
>
>namespace testABC123
#^^^^^^^^^ source.prim keyword.other.namespace.prim
#         ^ source.prim
#          ^^^^^^^^^^ source.prim entity.name.namespace.prim
>